### ISSUE TITLE
Zutilo hides the Zotfile Extract Annotations menu item

### ISSUE THREAD

**Issue by Servinjesus1** [wshanks/Zutilo#228]
It appears Zutilo makes the menu item for extracting annotations from PDFs by Zotfile disappear, requiring users to set a shortcut. ([see this bug](https://github.com/jlegewie/zotfile/issues/605)). Is this a new bug?

**Comment by qqobb** [comment]
> It appears Zutilo makes the menu item for extracting annotations from PDFs by Zotfile disappear

No, this has been removed from the context menu in ZotFile 5.1, which also provides Zotero 6 compatibility. See here:

https://forums.zotero.org/discussion/94128/zotfile-no-option-to-extract-annotations/p1

https://www.zotero.org/support/kb/zotfile_extract_annotations

> **Where did the Extract Annotations button go?**
> 
> “Extract Annotations” was a feature of the third-party ZotFile plugin. In Zotero 6, that feature has been replaced by Zotero's own advanced PDF functionality.

**Comment by wshanks** [comment]
Even in a clean profile where I just install ZotFile, I see Extract Attachments appear initially and then not appear when I open the context menu again, so maybe this is not Zutilo-specific?

**Comment by wshanks** [comment]
@qqobb do you think Zutilo should do something about the migration of this feature? Should the ZotFile shortcut start calling the built in Zotero function instead? Or should it be left alone and a new shortcut be added for the built in function? It seems like this shortcut is one that gets some use.

**Comment by qqobb** [comment]
@wshanks: I would suggest to keep the ZotFile shortcut for the time being as a legacy option. Some users might still rely on it because they haven't transitioned yet to Zotero's option. It also seems that ZotFile's annotation extraction function is still being used in its tablet functions. The fact that ZotFile's annotation extraction feature is hidden now in the context menu should've alerted users that something has changed.

We could consider adding a new shortcut for the corresponding Zotero function, perhaps with a new category "PDF reader" (#221). It's probably this function: [createNoteFromAnnotationsFromSelected](https://github.com/zotero/zotero/search?q=createNoteFromAnnotationsFromSelected). But Zotero's internal PDF reader already offers various ways of dealing with annotations, and there are various workflows now that are not based on extracting all annotations. For example, you can drag and drop, or copy (Ctrl+C) and paste (Ctrl+V), annotations from the annotation side bar to a Zotero note. If you paste to a text editor, you get a Markdown version with zotero:// links. The shortcut might still be useful for users of external PDF editors, though.

**Comment by adzcai** [comment]
Could we reopen this issue? This would be a helpful shortcut to include. It'd save some time from having to open the PDF, select all annotations, create a new note, and paste the annotations in. I'd be happy to add this shortcut.
